pkg/scheduler/uthelper/helper.go

```diff
@@ -156,11 +151,19 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
 		}
 	}
 
-	if len(test.Bind) != len(binder.Binds) {
-		return fmt.Errorf("case %d(%s) check bind: \nwant: %v\n got %v ", caseIndex, test.Name, test.Bind, binder.Binds)
+	// in case expected test.BindsNum is 0, but actually there is a binding and wait the binding goroutine to run
```

Comment:
In which condition will this happen?

Reply:
> In which condition will this happen?

The main process is quit soon if test.BindsNum is 0 which means there is no expected binding. As the binding is aync goroutine, and it need time to be scheduled and run. So we need to wait it is started.